gammapy/obs/data_store.py

```diff
@@ -99,6 +100,25 @@ def from_name(cls, name):
         dm = DataManager()
         return dm[name]
 
+    @classmethod
+    def from_all(cls, val):
+        """Try different DataStore construtors.
+        
+        Currently tried
+        - from_dir
```

Comment:
Format this somehow that links to the other methods are created in the HTML output?